.drone.yml

```diff
@@ -10,4 +10,4 @@ build:
     - ./bootstrap.sh -c $$COMMIT -p sg
     - ./build.sh -v
     - ./test.sh
-    - ./test.sh -race
+    - ./test.sh -race -timeout 20m
```

Comment:
Is this change required for the new test in this PR, or a change that's just generally needed?

Reply:
@adamcfraser It's not a requirement for the test included in this PR, but something is causing us to just tip over the default 10m timeout on drone. Could be a combination of things. I'll pull this out into a separate PR if nessesary.